tests/performance_tests.py

```
@@ -37,3 +42,63 @@
+
+    async with KuboCAS(
+        rpc_base_url=rpc_base_url, gateway_base_url=gateway_base_url, headers=headers
+    ) as kubo_cas:
+
+        root_cid = "bafyr4ialorauxcpw77mgmnyoeptn4g4zkqdqhtsobff4v76rllvd3m6cqi"
+        # root_node_id = CID.decode(root_cid)
+
+        hamt = await HAMT.build(
+            cas=kubo_cas, root_node_id=root_cid, values_are_bytes=True, read_only=True
+        )
+        start = time.perf_counter()
```

Ruff (0.11.9)

Local variable `start` is assigned to but never used

Remove assignment to unused variable `start`

(F841)